test/functional/sendheaders.py

```diff
@@ -339,16 +356,16 @@ def test_nonnull_locators(self, test_node, inv_node):
                     # getdata requests (the check is further down)
                     inv_node.send_header_for_blocks(blocks)
                     inv_node.sync_with_ping()
-                [ test_node.send_message(msg_block(x)) for x in blocks ]
+                [test_node.send_message(msg_block(x)) for x in blocks]
                 test_node.sync_with_ping()
                 inv_node.sync_with_ping()
                 # This block should not be announced to the inv node (since it also
                 # broadcast it)
```

Comment:
nit: test node, not inv node, broadcasts the block.

Reply:
In this section of the test, both `test_node` and `inv_node` announce the new block.